### DISCUSSION TITLE
Avoid serving legacy JavaScript to modern browsers / Serve static assets with an efficient cache policy

### DISCUSSION THREAD

**Discussion by bycixman35** [nuxt/nuxt#20224] [Questions]
Hello everyone, I'm coding a project using nuxt 3 and typescript, but I'm getting 2 errors in my project and I couldn't solve them no matter what I did. I will be grateful if you could help me.

First Error:  Avoid serving legacy JavaScript to modern browsers 

Second Error:  Serve static assets with an efficient cache policy

**Comment by danishasifdev** [comment]
I came here for an answer :(
